source/GameData.cpp

```diff
@@ -180,8 +182,11 @@ bool GameData::BeginLoad(const char * const *argv)
 			LoadFile(path, debugMode);
 	}
 	
-	// Now that all the stars are loaded, update the neighbor lists.
-	UpdateNeighbors();
+	// Now that all the stars are loaded, update the neighbor lists and other
```

Comment:
The ships and outfits also need to be loaded at this point (since we get the neighbor distances from the jumpdrive outfits and jumpdrives built-in into ships).
I would expect they are.

Reply:
They are (everything is loaded at this point), but I’ll update the comment.